src/components/atoms/SfAlert/SfAlert.html

```diff
@@ -0,0 +1,19 @@
+<div class="sf-alert" :class="`sf-alert--${type}`">
+    <slot>
+        <slot name="icon" :icon="icon" v-if="icon">
+            <div class="sf-alert__icon" >
+               <img :src="icon" v-if="typeof icon === 'string'">
+               <template v-else>
+                    <img src="../../../assets/alert-alert.svg" v-if="type === 'alert'">
```

Comment:
`<img :src="'../../../assets/alert-' +  type  + '.svg'">` except of this 3 checks is also enough, the same way you did with `class`

Reply:
it's because of webpack hashing names of the images, it's the only way it works.